benchs/bench-consumers/src/runners/runner_all.rs

```diff
@@ -14,13 +13,7 @@ use super::{
 #[allow(dead_code)]
 pub async fn run_all_benchmarks() -> Result<()> {
     let use_nats_compression = false; // adjust as needed
-    let mut nats = NatsHelper::connect(use_nats_compression).await?;
-    nats.data_parser_mut()
-        .set_serialization_type(SerializationType::Postcard); // adjust as needed
-    nats.data_parser_mut()
-        .set_compression_type(CompressionType::Zlib); // adjust as needed
-    nats.data_parser_mut()
-        .set_compression_level(CompressionLevel::Fastest); // adjust as needed
+    let nats = NatsHelper::connect(use_nats_compression).await?;
```

Comment:
We can adjust these parameters statically. Please see `connect` fn impl for details